beta/vercel.json

```diff
@@ -254,5 +254,43 @@
       "destination": "https://5a046bf5a6188f4b8fa4938a--reactjs.netlify.com",
       "permanent": false
     }
+  ],
+  "headers": [
+    {
+      "source": "/fonts/SourceCodePro-Regular.ttf",
```

Comment:
There should probably be a way to add all font files inside the `fonts` like `/fonts/(.*).ttf` ?

Reply:
Thanks. I've improved the syntax.